src/react-components/room/AvatarSettingsContent.scss

```diff
@@ -16,8 +16,8 @@
   /* TODO: This styling into AvatarPreview */
   & > :first-child {
     width: 168px;
-    height: 300px;
-    min-height: 300px;
+    height: 250px;
```

Comment:
I don't think height is required here since min-height should be overwriting it AFAIK.

Reply:
Ah, good call. I'll double-check if I can remove that safely.

Reply:
Turns out that `height` was the required property here. It's a bit of quirk, since the `AvatarPreview` component that is contained by this component resizes itself to its container's dimensions. Removed min-height and added a comment to explain that.